Assets/Scripts/Player/PlayerView.cs

```diff
@@ -19,8 +20,11 @@ public GameObject BulletSpawnPos
             get { return bulletSpawnPos; }
         }
 
-        public void SetController(PlayerController playerController)
+        private int playerHealth = 0;
```

Comment:
health should be in the model and not stored in view

Reply:
Done.